wordpress-mobile/WordPress-Android
# Create a debug screen for displaying all shared pref flags

Created a screen for internal purposes to display and modify saved local flags. It's available on the debug settings screen

_PS: I'm open to a better name for the screen_ 😄 

-----

## To Test:

- Launch a debug app
- Go to Me -> Debug settings
- Open a dropdown menu on the toolbar and tap on `All local flags`
- Make sure you can see your locally saved boolean flags from shared prefs
- Modify any and make sure the new value is saved

-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

3. What automated tests I added (or what prevented me from doing so)

    - N/A

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
